dask/dataframe/tests/test_rolling.py

```diff
@@ -111,7 +109,9 @@ def test_rolling_funtions_dataframe():
 ])
 @pytest.mark.parametrize('window', [1, 2, 4, 5])
 @pytest.mark.parametrize('center', [True, False])
-@pytest.mark.parametrize('axis', [0, 1, 'rows', 'columns'])
+@pytest.mark.parametrize('axis', [0, 'columns'])
+@pytest.mark.skipif(LooseVersion(pd.__version__) < '0.18.0',
+                    reason="rolling object not supported")
```

Comment:
Thanks for the attention and thanks also for debugging this code, I didn't get a chance to look at it during the week.

FWIW, the referenced issue above was that for the `npartitions = 2, method = 'kurt', args = (), window = 4, center = True, axis = 0` case I got a failure `AssertionError: expected -0.11206 but got -0.11206, with decimal 5`. I can't seem to reproduce it, though.